app/src/main/java/pl/org/akai/audioai/screens/LoginScreen.kt

```diff
@@ -1,31 +1,147 @@
-        Text("LoginScreen")
-        Spacer(modifier = Modifier.height(8.dp))
-        Button(onClick = {
-            navController.navigate(Screen.AssistantScreen.route)
-        },
-            modifier = Modifier.align(Alignment.End)
-        ) {
-            Text(text = "To AssistantScreen")
+    // Chyba button Login będzie odpowiadał za navi do AssistantScreena
+//    Column(
+//        modifier = Modifier
+//            .padding(20.dp)
+//            .fillMaxHeight(),
+//        verticalArrangement = Arrangement.Bottom,
+//        horizontalAlignment = Alignment.CenterHorizontally
+//    ) {
+//        Spacer(modifier = Modifier.height(8.dp))
+//        Button(
+//            onClick = { navController.navigate(Screen.AssistantScreen.route) },
+//            modifier = Modifier.align(Alignment.End)
+//        ) {
+//            Text(text = "To AssistantScreen")
+//        }
+//    }
+
+    Column(
+        modifier = Modifier
+            .padding(50.dp),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+
+        val context = LocalContext.current
+        val username = remember { mutableStateOf("") }
+        val password = remember { mutableStateOf("") }
+
+        Spacer(modifier = Modifier.height(100.dp))
+        Text(text = "Login", style = TextStyle(fontSize = 40.sp))
+
+        Spacer(modifier = Modifier.height(20.dp))
+        TextField(
+            label = { Text(text = "Username") },
+            value = username.value,
+            onValueChange = { username.value = it })
+
+        Spacer(modifier = Modifier.height(20.dp))
+        TextField(
+            label = { Text(text = "Password") },
+            value = password.value,
+            visualTransformation = PasswordVisualTransformation(),
+            keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Password),
+            onValueChange = { password.value = it })
+
+        Spacer(modifier = Modifier.height(20.dp))
+        Box(modifier = Modifier.padding(40.dp, 0.dp, 40.dp, 0.dp)) {
+            Button(
+                onClick = {
+                    if(username.value.isEmpty() and password.value.isNotEmpty()){
+                        Toast.makeText(context, "Username is Empty", Toast.LENGTH_SHORT).show()
+                    }
+                    if (username.value.isEmpty() and password.value.isNotEmpty()){
+                        Toast.makeText(context, "Password is Empty", Toast.LENGTH_SHORT).show()
+                    }
+                    if(username.value.isEmpty() and password.value.isEmpty()){
+                        Toast.makeText(context, "Username and Password are Empty", Toast.LENGTH_SHORT).show()
+                    }
+                    if(username.value.isNotEmpty() and password.value.isNotEmpty()){
+                        val user = true //until we have no authorization
+                        if (user)
```

Comment:
nitpick: formatting is a little bit off. You can use autoformatter with `ctrl + alt + l)`